dq-game/src/client/event_coordinator.py

```diff
@@ -16,6 +16,9 @@ def on_timeout(self):
         self.renderer.screen_handler.clear_data()
         self.renderer.show_logo()
 
+    def on_answer_limit_reached(self):
+        self.renderer.screen_handler.clear_data()
```

Comment:
When the limit of answer is reached, clear the data (callback and buttons) but show logo also ? Or a timeout message ?